jenkins-config/clouds/openstack/Primary/ubuntu18.04-docker-4c-16g.cfg

```diff
@@ -0,0 +1,2 @@
+IMAGE_NAME=ZZCI - Ubuntu 18.04 - docker - x86_64 - 20200925-233259.113
+HARDWARE_ID=lf-standard-4
```

Comment:
@tonyespy I think this should be changed to use the v2 High CPU flavor node. I did a bunch of testing on x86 nodes and found this to be the most performant for Snaps.

The change would look like this:
```
HARDWARE_ID=v2-highcpu-8
VOLUME_SIZE=40
```

Reply:
@ernestojeda thanks for feedback, updated and re-pushed. Is the name of the file still OK with this change?

Reply:
For a v2-highcpu-8, the file named should be changed to "8c-8g".

Reply:
@tonyespy I created this node on the sandbox and then created a simple pipeline to run a test and it seems to work. https://jenkins.edgexfoundry.org/sandbox/view/All/job/Functional-Testing/job/ubuntu-node-test/2/console. Node label is `ubuntu18.04-docker-8c-8g`